x/profiles/client/cli/cli_app_links.go

```diff
@@ -111,7 +111,7 @@ func GetCmdUnlinkApplication() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "unlink-app [application] [username]",
 		Short:   "Unlink a centralized application account from your Desmos profile",
-		Example: fmt.Sprintf("%s tx profiles unlink-app [application] [username]", version.AppName),
+		Example: fmt.Sprintf(`%s tx profiles unlink-app "twitter" "twitter_user" --from alice`, version.AppName),
```

Comment:
We wither specify the `--from` flag for all transaction commands, or for none of them
```suggestion
		Example: fmt.Sprintf(`%s tx profiles unlink-app "twitter" "twitter_user"`, version.AppName),
```